js/mailalerts.js

```diff
@@ -33,20 +33,39 @@ function  addNotification() {
     success: function (resp) {
       resp = JSON.parse(resp);
 
-      $('div.js-mailalert > span').html('<article class="alert alert-info" role="alert" data-alert="success">'+resp.message+'</article>').show();
+      const alertProperties = {
+        class: resp.error ? 'danger' : 'success',
+        data: resp.error ? 'error' : 'success'
+      }
+
+      $('div.js-mailalert > span').html('<article class="mt-1 alert alert-' + alertProperties.class + '" role="alert" data-alert="' + alertProperties.data + '">'+resp.message+'</article>').show();
       if (!resp.error) {
         $('div.js-mailalert > button').hide();
         $('div.js-mailalert > input[type=email]').hide();
-        $('div.js-mailalert > #gdpr_consent').hide();
+        $('div.js-mailalert .gdpr_consent_wrapper').hide();
       }
     }
   });
   return false;
 }
 
-$('document').ready(function()
-{
-  $('.js-remove-email-alert').click(function()
+$(document).on('ready', function() {
+  const mailAlertWrapper = $('.js-mailalert');
+  const mailAlertSubmitButton = mailAlertWrapper.find('button');
+
+  if (mailAlertWrapper.find('#gdpr_consent').length) {
+    setTimeout(() => {
```

Comment:
I don't understand the reason behind the timeout :thinking: 
Also why this timeout is registering the event onChange ? :thinking:

Reply:
I suggest trying this code without a timeout, it's impossible to work with for example `.off()`, we need to make sure that our code is executed **after** GDPR module, this is all done this way to avoid changes in GDPR module

Reply:
maybe @NeOMakinG has a better idea, without touching GDPR module, but timeout is very common as a workaround 🤷

Reply:
Can't we deal with JS priority, or module priority or whatever ?

Overwise it's fine for me, but dealing with callstack this way is pretty hard to maintains, if multiple modules does the same thing, we've no way to find which one is executed before which one

Reply:
If we've no solution for this except than tricking the callstack, then at least add a comment

```suggestion
    // We use a timeout to put this at the end of the callstack, so it's executed after GPDR module. 
    setTimeout(() => {
```